oauth/templates/oauth2_provider/authorize.html

```diff
@@ -0,0 +1,44 @@
+{% extends "oauth2_provider/base.html" %}
+
+{% load i18n %}
+{% block content %}
+    <div class="block-center">
+        {% if not error %}
+            <form id="authorizationForm" method="post">
+                <h3 class="block-center-heading">{% trans "Authorize" %} {{ application.name }}?</h3>
+                {% csrf_token %}
+
+                {% for field in form %}
+                    {% if field.is_hidden %}
+                        {{ field }}
+                    {% endif %}
+                {% endfor %}
+
+                {% if scopes_descriptions %}
+                  <p>{% trans "Application requires following permissions:" %}</p>
+                  <ul>
+                      {% for scope in scopes_descriptions %}
+                          <li>{{ scope }}</li>
+                      {% endfor %}
+                  </ul>
+                {% else %}
+                  <p>The application does not require any specific permission.</p>
+                {% endif %}
+
+                {{ form.errors }}
+                {{ form.non_field_errors }}
+
+                <div class="control-group">
+                    <div class="controls">
+                        <input type="submit" class="btn btn-large" value="Cancel"/>
```

Comment:
All this code was only a copy paste from the native app's template.
I'm not taking care of the translations right now as it would mean collecting each place that is not translated in the apps.
However Ybon was making the same notification.